mitmproxy/flow/modules.py

```diff
@@ -287,66 +284,3 @@ def next_flow(self, request):
             return l[0]
         else:
             return l.pop(0)
-
-
-class StickyCookieState:
-    def __init__(self, flt):
-        """
-            flt: Compiled filter.
-        """
-        self.jar = collections.defaultdict(dict)
-        self.flt = flt
-
-    def ckey(self, attrs, f):
-        """
-            Returns a (domain, port, path) tuple.
-        """
-        domain = f.request.host
-        path = "/"
-        if "domain" in attrs:
-            domain = attrs["domain"]
-        if "path" in attrs:
-            path = attrs["path"]
-        return (domain, f.request.port, path)
-
-    def domain_match(self, a, b):
-        if http_cookiejar.domain_match(a, b):
-            return True
-        elif http_cookiejar.domain_match(a, b.strip(".")):
-            return True
-        return False
-
-    def handle_response(self, f):
-        for name, (value, attrs) in f.response.cookies.items(multi=True):
-            # FIXME: We now know that Cookie.py screws up some cookies with
-            # valid RFC 822/1123 datetime specifications for expiry. Sigh.
-            dom_port_path = self.ckey(attrs, f)
-
-            if self.domain_match(f.request.host, dom_port_path[0]):
-                if cookies.is_expired(attrs):
```

Comment:
Deletion of cookies was recently added in #1324 (fixing #1096) and has now been removed. :confused: